src/Computer/Settings.hpp

```diff
@@ -122,6 +123,23 @@ struct PlacesOfInterestSettings {
   void SetHome(const Waypoint &wp);
 };
 
+/**
+ * Options for radio remote control
+ */
+struct RadioSettings {
+  RadioFrequency active_frequency;
+  RadioFrequency standby_frequency;
+
+  StaticString<32> active_name;
+  StaticString<32> standby_name;
+
+  void SetDefaults() {
+    active_frequency.Clear();
+    standby_frequency.Clear();
+    active_name = "";
```

Comment:
Use `clear()`, which sets one byte; this line calls `strcpy()` which is unnecessary overhead.

Reply:
Used clear()